scriptmodules/system.sh

```diff
@@ -74,6 +74,17 @@ function platform_rpi() {
     __binary_url="http://downloads.petrockblock.com/retropiebinaries/"
 }
 
+function platform_rpi2() {
+    __default_cflags="-O2 -mfpu=neon -march=armv7-a -mfloat-abi=hard"
+    __default_asflags=""
+    __default_makeflags=""
+    __default_gcc_version="4.7"
+    __qemu_cpu=cortex-a8
```

Comment:
Well, to compile for the rpi2 from a pi (or in a chroot), you would just do 

```
sudo __platform=rpi2 ./retropie_setup.sh
```

I have built a binary set for the rpi2, which i can upload and users can try and see if there is any noticable performance improvement.

if cross compiling with distcc, you shouldn't have to do any more that the adjusted commandline above, and add any additional makefile options etc such as pathing in distcc binaries.